### PR TITLE
various fixes: keyboardavoidingview, blur text input, asc|desc UI rev…

### PR DESCRIPTION
**Pull request by valpinkman**
…ersed, currency icons in accountlist

### CODE REVIEW

File: src/components/OperationRow.js [hunk 1]
```diff
@@ -56,6 +56,7 @@ class OperationRow extends PureComponent<Props, *> {
               ellipsizeMode="tail"
               style={[styles.topLine, { color: valueColor }]}
             >
+              {isIn ? "+ " : "- "}
```
Comment (gre) [inline src/components/OperationRow.js]: This shouldn't be done like this, in fact it's part of formatCurrencyUnit to do this (there is utilities in live-common to have the actual value from the operation + a special mode to force display of "+")

Idea of this is that potentially the way this is formatted depends on the locale

File: src/screens/SelectFunds/index.js [hunk 2]
```diff
@@ -90,67 +98,74 @@ class SelectFunds extends Component<Props, State> {
     const isWithinBalance = amountBigNumber.lt(account.balance);
     const isValid = amountBigNumber.gt(0) && isWithinBalance;
 
+    const keyboardVerticalOffset = Platform.OS === "ios" ? 60 : 0;
```
Comment (gre) [inline src/screens/SelectFunds/index.js]: Interesting. Any idea why 60 ?

Comment (valpinkman) [reply src/screens/SelectFunds/index.js]: because 40 wasn't enough

File: src/components/OperationRow.js (hunk 1 above)
Comment (valpinkman) [reply src/components/OperationRow.js]: ok I need to check this out then

**Pull request merged**